src/ui/public/agg_types/controls/precision.html

```diff
@@ -36,3 +36,11 @@
   </label>
 </div>
 
+<div class="vis-option-item indented">
+  <label>
+    <input type="checkbox"
+           name="useFilter"
+           ng-model="agg.params.useFilter">
+    Enable collar (geo_bounding_box filter)
```

Comment:
I had to shorten the description to get it to fit in the options width constraints. I set it to `Only request data around extent of map <kbn-info info="Apply geo_bounding_box filter aggregation to narrow the subject area to the map view box with collar."></kbn-info>`